blog/2024-05-16-community-days.mdx

```diff
@@ -36,6 +36,8 @@ in which you can get active and code on Tractus-X yourself with our support.
 
 Once again, a community party awaits you on the first evening for networking and talking shop with delicious food and an entertaining program.
 
+The Eclipse Tractus-X Community Days start on Thursday at 12 noon and end on Friday at around 4 p.m.
```

Comment:
I would suggest to use 12 p.m. (noon) and add the time zone (GMT+1 ?) as we expect international participants. 
Thanks!